### DISCUSSION TITLE
[Geochemistry Module] Kinetic Rate as an aux-variable

### DISCUSSION THREAD

**Discussion by batodon** [idaholab/moose#24584] [Q&A Modules: General]
Hello Moose Geochemistry Experts,

Is it possible to output the dissolution rate calculated by the `GeochemistryKineticRate` UO as an aux-variable? For example, let’s say I want to output the kinetically-controlled dissolution rate of quartz and then use this rate as a coupled variable in a PorousFlow material. Is this feasible, and how do I go about it? Thx.

**Comment by cpgr** [comment]
@WilkAndy?
